e2e/vmcluster_test.go

```diff
@@ -137,6 +137,28 @@ var _ = Describe("e2e vmcluster", func() {
 				Eventually(func() string {
 					return expectPodCount(k8sClient, 2, namespace, vmCluster.VMInsertSelectorLabels())
 				}, 70, 1).Should(BeEmpty())
+
+				By("vmInsert revisionHistoryLimit update to 3")
```

Comment:
minor typo here)
"vmInsert revisionHistoryLimit update to 3" -> "vminsert revisionHistoryLimit update to 2"

Reply:
Done